_includes/v22.2/start-in-docker/mac-linux-steps-single-node.md

```diff
@@ -0,0 +1,130 @@
+When you use the `cockroach start-single-node` command to start a single-node cluster with Docker, additional features are available to help with testing and development.
+
+{{site.data.alerts.callout_danger}}
+Single-node clusters are not highly available or fault tolerant. They are not appropriate for production use.
+{{site.data.alerts.end}}
+
+- You can optionally set the following [Docker environment variables](https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file) to create a database and user automatically and to set a password for the user.
+  - `COCKROACH_DATABASE`
+  - `COCKROACH_USER`
+  - `COCKROACH_PASSWORD`
+
+      To prevent loss of a cluster's existing data, the environment variables are used only if the `/cockroach/cockroach-data` directory within the container is empty.
+
+- You can optionally mount a directory of initialization scripts into the `docker-entrypoint-initdb.d` directory within the container. These scripts are run after CockroachDB starts and after the database and user (if specified as environment variables) have been created. The scripts run in the alphanumeric sort order imposed by your locale. The init scripts are run only if the `/cockroach/cockroach-data` directory within the container is empty.
+
+During local development and testing, you can re-initialize the default database, user, and password by deleting the contents of `/cockroach/cockroach-data` within the running container and then restarting the container.
+
+This section shows how to start a single-node cluster that uses these features.
+
+### Step 1. Create a Docker volume for the node
+
+Cockroach Labs recommends that you store cluster data in a Docker volume rather than in the storage layer of the running container. Otherwise, if a Docker container is inadvertently deleted, its data is inaccessible.
+
+To create the [Docker volume](https://docs.docker.com/storage/volumes/) where the cluster will store its data, run the following:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+docker volume create roach-single
+~~~
+
+### Step 2. Start the cluster
+
+{% capture  env_file_note %}{{site.data.alerts.callout_success}}Instead of specifying each value directly by using the `-e` or `--env` flag, you can store them in a file on the Docker host. Use one key-value pair per line and set the `--env-file` flag to the file's path.{{site.data.alerts.end}}{% endcapture %}
+
+1. Start the cluster node.
+
+    The following command starts a single-node cluster that:
```

Comment:
The command that is "following" is so far away because all the conditions come before it.  Here's an alternative.

To start a cluster that:

bullets

run the following command:

code block

Also it's odd that you are presenting a tip in the middle of the bullets for an alternate way to set the env variables, but then the code snippet disregards that tip. Why not put the tip at the very end of either the bullets or after the code block.  If you pulled the tip out from the bullets you might be able to go back to Markdown instead of HTML.

<img width="1016" alt="Screen Shot 2022-09-30 at 7 11 43 AM" src="https://user-images.githubusercontent.com/20482960/193289396-137aecef-2410-46e8-95e6-9258d0a1ef87.png">